subsys/zigbee/osif/zb_nrf_transceiver.c

```diff
@@ -23,21 +23,26 @@
 
 #if defined(NRF52840_XXAA) || defined(NRF52811_XXAA) || \
 defined(NRF5340_XXAA_NETWORK) || defined(NRF5340_XXAA_APPLICATION)
-/* dBm value corresponding to value 0 in the EDSAMPLE register. */
-#define ZBOSS_ED_MIN_DBM       (-92)
+/* Minimum value in dBm detectable by the radio. */
+#define MIN_RADIO_SENSITIVITY (-92)
 /* Factor needed to calculate the ED result based on the data from the RADIO peripheral. */
 #define ZBOSS_ED_RESULT_FACTOR 4
 
-#elif defined(NRF52833_XXAA) || defined(NRF52820_XXAA)
-/* dBm value corresponding to value 0 in the EDSAMPLE register. */
-#define ZBOSS_ED_MIN_DBM       (-93)
+#elif defined(NRF52833_XXAA) || defined(NRF52820_XXAA) \
+|| defined(NRF5340_XXAA_NETWORK) || defined(NRF5340_XXAA_APPLICATION)
```

Comment:
`NRF5340_XXAA_NETWORK` and `NRF5340_XXAA_APPLICATION` are mentioned twice ;)

Reply:
Deleted where unneeded